semperfiwebdesign/all-in-one-seo-pack
# Refactor main plugin file into a core class

Issue #1397

## Proposed changes

Refactors the main plugin file into a core class.

## Types of changes

What types of changes does your code introduce?

- Removing old code/functionality
- Improves existing functionality
- Improves existing code

## Checklist


## Testing instructions
* Check to see if plugin is functional when active.
* Check plugins page to see additional links are added.
* Test scan on Opengraph settings page.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...
